src/DoubleTheDonation/Helpers/DoubleTheDonationApi.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\Helpers;
+
+/**
+ * Helper class used for handling DTD credentials
+ *
+ * @unreleased
+ */
+class DoubleTheDonationApi
```

Comment:
Might be a little tricky but would be great if we had a unit test for this

Reply:
I'm not even sure how to test this